DataConnectors/Imperva WAF Gateway/Logo/Imperva_wordmark_1color_DarkGrey.svg

```diff
@@ -0,0 +1,15 @@
+    </defs>
+    <g id="Artboard" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="Imperva_wordmark_1color_DARK" transform="translate(115.500000, 113.000000)">
+            <mask id="mask-2" fill="white">
+                <use xlink:href="#path-1"></use>
+            </mask>
+            <use id="path-3" fill="#212121" xlink:href="#path-1"></use>
```

Comment:
Please correct the logo, remove Illegal attributes like xmlns:xlink and xlink:href. Id should have Guid to make the logo identifiable across all Azure logo assets. Please refer https://github.com/Azure/Azure-Sentinel/blob/e1665d4c4442b430d6847adc65f4b9c25f7b071f/DataConnectors/ReadMe.md for logo requirements.